service/worker/batcher/activities.go

```diff
@@ -626,7 +657,7 @@ func startTaskProcessorProtobuf(
 				err = processTask(ctx, limiter, task,
 					func(workflowID, runID string) error {
 						_, err := frontendClient.SignalWorkflowExecution(ctx, &workflowservice.SignalWorkflowExecutionRequest{
-							Namespace: batchOperation.Namespace,
+							Namespace: batchOperation.Request.Namespace,
```

Comment:
You should resolve the namespace using the ID via the namespace registry (that was the whole point of using the ID).

Reply:
If these run in the system namespace do we still have to do that? Can we pass in the activity default namespace?